lib/ui/painting.dart

```diff
@@ -1621,22 +1737,27 @@ typedef ImageDecoderCallback = void Function(Image result);
 ///
 /// To obtain an instance of the [FrameInfo] interface, see
 /// [Codec.getNextFrame].
-@pragma('vm:entry-point')
-class FrameInfo extends NativeFieldWrapperClass2 {
+///
+/// The recipient of this class is responsible for calling [Image.dispose] on
```

Comment:
Should we say "The recipient of this class, who obtained it from [Codec.getNextFrame], becomes the owner of it and is responsible for ..."

Should we also say that you shouldn't share the FrameInfo object with anybody else and instead hand out handles to the wrapped image?

Reply:
Part of what I'm struggling with on this one is that we don't have any clear request for sharing/reusing frame info. If we do, the right thing to do is add a `clone` method here. 

It's not so much that I'm opposed to adding that as that I'm reluctant to do it here without a clearer idea of why you would do it.

Reply:
I still think that this comment should get a little more details about the image handle it contains (or link to documentation that better described how to deal with it).

Reply:
I think `[Image.clone]` is the right place to link to for additional information.

I've added a couple code samples ("BAD" and "GOOD") to demonstrate usage.